ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh

```diff
@@ -8,6 +8,32 @@ echo "TEST_TYPE=${TEST_TYPE}"
 echo "BRANCH=${BRANCH}"
 echo "ARCH=${ARCH}"
 
+# Until the yellow-zone network bandwidth is upgraded, we will run the following tests in their own periodic and exclude
+# them from the conformance-parallel workflow.
+if [ "${BRANCH}" == "4.15" ] && [[ "${TEST_TYPE}" == "conformance-parallel" || "${TEST_TYPE}" == "heavy-build" ]]; then
+  cat > "${SHARED_DIR}/excluded_tests" << EOF
+"[sig-apps][Feature:DeploymentConfig] deploymentconfigs with multiple image change triggers should run a successful deployment with a trigger used by different containers [apigroup:apps.openshift.io][apigroup:image.openshift.io] [Skipped:Disconnected] [Suite:openshift/conformance/parallel]"
```

Comment:
isn't there like a `TEST_SKIPS` variable that accepts regex that we can configure in the workflow itself? that should eliminate needing to do this in the step itself

Reply:
I can look into that, but the list to skip isn't every test in those categories (I don't think its every `[Feature:Builds]` test or every `[Feature:Image]` test, etc), so using this explicit list makes sense to me.  Also, this is already being done for a few different `TEST_TYPE` definitions below, so I think it makes sense.  Also, this functionality lets me do the "cheeky" thing and use this same file of `excluded_tests` as the list of runnable tests in the `heavy-build` job I created to cover the running of these on their own.